bundles/org.openhab.binding.volumio/src/main/resources/OH-INF/i18n/volumio_de.properties

```diff
@@ -0,0 +1,15 @@
+#add-on
```

Comment:
This file should be removed and generated using Crowdin.

Reply:
resolved